olp-cpp-sdk-dataservice-read/src/repositories/ApiCacheRepository.cpp

```diff
@@ -28,6 +29,8 @@ std::string CreateKey(const std::string& hrn, const std::string& service,
 }
 }  // namespace
 
+#define ACR_LOGTAG "ApiCacheRepository"
```

Comment:
Instead of defines, I would suggest having constexpr char* in the anonymous namespace.